MainApp/views.py

```diff
@@ -159,7 +159,11 @@ def removeFollower(self,request):
         unfollow.delete()
 
         #reload the page and make sure an follow button shows back up
-        return HttpResponseRedirect(reverse('MainApp:profile', args=[request.POST['unfollow']]))
+        if arg == None:
+            arg = [request.POST['unfollow']]
```

Comment:
By default we would redirect to the user we just unfollowed...but this can be override with a different username if needed.